components/ntp_sponsored_images/regional_component_data.cc

```diff
@@ -0,0 +1,754 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ntp_sponsored_images/regional_component_data.h"
+
+// This list should be synced with the list of generateNTPSponsoredImages.js
+// and packageNTPSponsoredImagesComponents.js in brave-core-crx-packager.
```

Comment:
Kind of a future thought (although, not important if locales don't change often). We could update `brave-crx-packager` to have an npm task to generate the value that gets assigned to `regional_data` (so that you can copy/paste it here)

Would be a nice follow up to https://github.com/brave/brave-core-crx-packager/pull/79, to make sure there isn't any human error in punching them in

Reply:
Yup, good idea 👍